graph/update_places.go

```diff
@@ -1,28 +1,33 @@
 package graph
 
 import (
+	"github.com/sirupsen/logrus"
 	"github.com/weihesdlegend/Vacation-planner/POI"
 	"github.com/weihesdlegend/Vacation-planner/iowrappers"
-	"googlemaps.github.io/maps"
 	"strings"
 	"sync"
 )
 
 type PlaceDetailsResultMap struct {
 	sync.Mutex
-	results map[string]maps.PlaceDetailsResult
+	results map[string]POI.Place
 }
 
-func newPlaceDetailsResultMap() *PlaceDetailsResultMap {
-	return &PlaceDetailsResultMap{results: map[string]maps.PlaceDetailsResult{}}
+func detailsResultMap() *PlaceDetailsResultMap {
+	return &PlaceDetailsResultMap{results: map[string]POI.Place{}}
 }
 
 func placeNeedUpdate(place *POI.Place) bool {
-	return len(strings.TrimSpace(place.GetURL())) == 0 || place.GetURL() == iowrappers.GoogleSearchHomePageURL
+	if len(strings.TrimSpace(place.GetURL())) == 0 || place.GetURL() == iowrappers.GoogleSearchHomePageURL {
+		logrus.Debugf("place with ID: %s need URL update", place.GetID())
```

Comment:
Add a special label in the log for easy search [TEST REDIS URL UPDATE]

Reply:
Done